cuda/private/rules/cuda_library.bzl

```diff
@@ -141,4 +141,5 @@ cuda_library = rule(
     },
     fragments = ["cpp"],
     toolchains = use_cpp_toolchain() + use_cuda_toolchain(),
+    provides = [CcInfo],
```

Comment:
Is it better to provide all of them? That is `[DefaultInfo, OutputGroupInfo, CcInfo, CudaInfo]`?

Reply:
Thx,done.